fab/fabfile.py

```diff
@@ -359,14 +359,23 @@ def _confirm_translated():
 
 
 @task
-def setup_release():
+def setup_release(keep_days=0):
```

Comment:
Might be good to add `fab staging setup_release:keep_days=10` to a docstring here, I know I won't remember the syntax next time I want this.